src/test/kotlin/blackjack/BlackJackTest.kt

```diff
@@ -1,29 +1,25 @@
 package blackjack
```

Comment:
테스트 클래스는 테스트 대상 클래스와 같은 패키지에 위치하면 어떨까요?

```suggestion
package blackjack.model
```